experiments/dataSources/openAI/menuWithGPTActions/MenuWithGPTActions.xml

```diff
@@ -77,7 +77,7 @@
 						<field id="Subject" uniqueid="fi-1pHP-40790"/>
 						<field type="COMBO" label="Description (generated)" template="{{Description}}" redirectType="datasourcedefault" uniqueid="fi-1pIC-54772">
 							<interactions>
-								<interaction type="rightclick">
+								<interaction type="rightclick" suppressContextMenu="true">
```

Comment:
While you are fixing this PR - would you mind updating this example so that its a real Field,  and not just a template?   This is what we changed for EU2...

Reply:
Implemented